src/main/java/io/vertx/mqtt/impl/MqttClientConnection.java

```diff
@@ -239,4 +240,5 @@ void close(){
   void writeMessage(MqttMessage mqttMessage) {
     so.writeMessage(mqttMessage);
   }
+
```

Comment:
If you remove the above import and this blank line you can just revert this file to the original one because there are no changes.